R-package/R/lgb.Booster.R

```diff
@@ -49,7 +49,11 @@ Booster <- R6Class(
       }
       class(handle)     <- "lgb.Booster.handle"
       private$handle    <- handle
-      private$num_class <- 1L
+      if ("num_class" %in% names(params)){
+	private$num_class <- params$num_class
+      } else{
+	private$num_class <- 1L
+      }
       private$num_class <-
         lgb.call("LGBM_BoosterGetNumClasses_R", ret = private$num_class, private$handle)
```

Comment:
Something does not work as expected. Why private$num_class <- 1L only changing it to the code I wrote solves the problem. I don't know what the C function GetNumClasses exactly does. What I am sure is that setting the parameter num_class to the number of classes in the call of the lgb.train does not work.